sycl/source/handler.cpp

```diff
@@ -179,10 +180,10 @@ event handler::finalize() {
       // Make sure implicit non-interop kernel bundles have the kernel
       if (!KernelBundleImpPtr->isInterop() &&
           !MImpl->isStateExplicitKernelBundle()) {
+        auto Dev = (MGraph) ? MGraph->getDevice() : MQueue->get_device();
```

Comment:
```suggestion
        auto Dev = MGraph ? MGraph->getDevice() : MQueue->get_device();
```
Nit but the parenthesis are unnecessary here.